src/sandstorm/web-session.capnp

```diff
@@ -111,6 +147,48 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
     encoding @2 :Text;  # Content-Encoding header (optional).
   }
 
+  struct PropfindContent {
```

Comment:
Though I suppose having a single `dav()` method would be problematic for special handling of the "destination" header as mentioned above, and possibly other DAV headers which you aren't supporting yet but which we may want to support in the future... so maybe not.